### ISSUE TITLE
keep the reports offline

### ISSUE THREAD

**Issue by Vaillus** [mcndt/obsidian-toggl-integration#138]
It would be great if it was possible to keep the reports offline once they are loaded. 
thanks for the plugin, it is amazing !

**Comment by VincentX33** [comment]
Yes, if there is a way to store once loaded reports beyond a particular date, it would make life easier. What format is the rendered text getting stored as in the pre-render? Because only our :
```toggl
xxxx
```
stuff is getting stored in the file, i suppose the extension does the hard work of querying the toggl api every time we open a particular page!
